### PR TITLE
Linting: unused variable = error

### PR DESCRIPTION
**Pull request by ap-justin**
## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/components/RichText/RichText.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function RichText(props: Props) {
 
     try {
       quill.setContents(JSON.parse(props.content));
-    } catch (err) {
+    } catch (_) {
```
Comment (ap-justin) [inline src/components/RichText/RichText.tsx:26]: explictly not used should be named `_`

File: src/pages/Gift/Purchase/Purchaser/index.tsx [hunk 2]
```diff
@@ -8,8 +8,8 @@ import { schema } from "./schema";
 import { FormValues } from "./types";
 
 export default function Purchaser({
-  classes = "",
-  wallet,
+  classes: _c = "",
+  wallet: _w,
   ...state
 }: WithWallet<FormStep> & { classes?: string }) {
```
Comment (misicnenad) [inline src/pages/Gift/Purchase/Purchaser/index.tsx:14]:
```suggestion
}: FormStep) {
```
nit: we could update the props type and remove the need to even access `classes + wallet`

**Review by misicnenad — COMMENTED** [review]
Just one nit, otherwise looks good

File: src/pages/Gift/Purchase/Purchaser/index.tsx (hunk 2 above)
Comment (ap-justin) [reply src/pages/Gift/Purchase/Purchaser/index.tsx:14]: I tried this but would need to update parent of `Purchaser`, gift card is stale feature so just keep it as is for now so not to lose context when need to comeback

**Review by misicnenad — APPROVED** [review]

**Pull request merged**